### ISSUE TITLE
404 - Not Found - Template not found: vizy

### ISSUE THREAD

**Issue by strandofgenius** [verbb/vizy#177]
### Describe the bug

I was checking to see what kind of settings were available for Vizy and landed on a 404 error page. My guess is nothing needs to exist when you click "Vizy" under settings in the breadcrumbs of the Craft CP. Maybe just needs to redirect to the Vizy settings?

<img width="1007" alt="Screen Shot 2022-10-05 at 2 43 38 PM" src="https://user-images.githubusercontent.com/11370639/194148702-779fec33-70bb-40b4-9777-abbad6d105d6.png">

### Steps to reproduce

1. Install Craft CMS 4
2. Install Vizy
3. Go to CP Settings > Vizy
4. Click "Vizy" in the top breadcrumbs

### Craft CMS version

4.2.5.2

### Plugin version

2.0.7

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-4 as 2.0.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.8](https://github.com/verbb/vizy/releases/tag/2.0.8)
